src/Etats/menu.ts

```diff
@@ -21,9 +21,9 @@ export const Explain = {
         this.choix_diff = [];
         difficultes(this, null);
 
-        if (isNaN(Stats.diff)) Stats.diff = 1;
+        if (isNaN(Stats.level)) Stats.level = 1;
```

Comment:
I'm not sure about `level`. I thought `diff` mean `difficulty` 🤔 even if we have rename `level` in `stage` maybe we could keep `difficulty` as name ?